contents/docs/experiments/data-warehouse.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Data Warehouse
```

Comment:
Just to make this a bit more searchable and distinct from people looking for the data warehouse. Nav can stay the same though.
```suggestion
title: Using data warehouse tables in experiments
```